aws_instance.tf

```diff
@@ -5,8 +5,13 @@
 resource "aws_instance" "instanceWithNoVpc" {
   ami           = "some-id"
   instance_type = "t2.micro"
-  
+
```

Comment:
**Accurics remediation**
 
**[HIGH]: EC2 Instances are vulnerable to Server side request forgery**
 
**Vulnerability**
Ensure that EC2 instances within have IMDv1 disabled.
 
**Remediation**
IMDv1 is vulnerable to SSRF attack. Thus, EC2 Instance should have metadata_options block with either http_endpoint attribute set to disabled or ensure http_tokens attribute is set to required. This ensures IMDv2 is being used.

source: accurics-